drive/modules/listener/watch_schedular.bal

```diff
@@ -59,16 +59,17 @@ class Job {
     public isolated function execute() {
         error? err = self.registerWatchChannel();
         if (err is error) {
-            if (self.retryCount == 1){
+            if (self.retryCount == 1 && err.message().includes(ERR_UNAUTHORIZED_WEBHOOK_CHANNEL)){
                 log:printInfo(CHANNEL_REGISTRATION_IN_PROGRESS 
                 + " Waiting for " + self.domainVerificationDelay.toString() 
                 + " seconds to check result.");
                 log:printInfo(REQUEST_DOMAIN_VERIFICATION);
                 runtime:sleep(<decimal>self.domainVerificationDelay);
                 self.retryCount += 1;
                 self.execute();
-            } 
-            else if (self.retryCount <= self.retryMaxCount) {
+            } else if (err.message().includes(ERR_FILE_NOT_FOUND)){
+                panic error(ERR_INVALID_FILE_OR_FOLDER, 'error = err);
```

Comment:
Do we ask customer to input a file? No, right? When he see this error, he will wonder where has he pointed to a file.

Reply:
This particular error[1] comes when the user has entered a wrong file or folder ID[2] in listener configuration.

[1] `const string ERR_INVALID_FILE_OR_FOLDER ="Invalid folder or file !!";`
[2] ```public type ListenerConfiguration record {
    @display{label: "Port"}
    int port;
    @display{label: "Callback URL"}
    string callbackURL;
    @display{label: "Domain Verification File Content"}
    string domainVerificationFileContent;
    Configuration clientConfiguration;
    @display{label: "Specific Folder ID"}
    string? specificFolderOrFileId = ();
    ChannelRenewalConfig channelRenewalConfig?;
};
```